shared/chat/conversation/header.desktop.js

```diff
@@ -9,7 +9,7 @@ import type {Props} from './header'
 const Header = ({participants, onOpenFolder, onToggleSidePanel, sidePanelOpen, you, metaDataMap, followingMap, onShowProfile}: Props) => (
   <Box style={containerStyle}>
     <Usernames colorFollowing={true} inline={false} commaColor={globalColors.black_40} type='BodyBig' users={usernamesToUserListItem(participantFilter(participants, you).toArray(), you, metaDataMap, followingMap)}
-      containerStyle={{flex: 1, textAlign: 'center', justifyContent: 'center'}} onUsernameClicked={onShowProfile} />
+      containerStyle={{flex: 1, textAlign: 'center', justifyContent: 'center', marginLeft: 48}} onUsernameClicked={onShowProfile} />
```

Comment:
Hmm, no globalMargins.large is 40px, not 48. This tweak is to recenter the usernames in the header by pushing them by 2 times 24 (from both icons on the right side).